src/stakeinput.h

```diff
@@ -35,17 +36,17 @@ class CStakeInput
 class CPivStake : public CStakeInput
 {
 private:
-    Optional<CTxOut> opOutputFrom{nullopt};
-    Optional<COutPoint> opOutpointFrom{nullopt};
+    const CTxOut opOutputFrom;
+    const COutPoint opOutpointFrom;
```

Comment:
since these are no longer optionals,  please drop the `op` from the name, which is just confusing.

Reply:
done